sabnzbd/deobfuscate_filenames.py

```diff
@@ -230,6 +230,41 @@ def deobfuscate(nzo, filelist: List[str], usefulname: str):
         nzo.set_unpack_info("Deobfuscate", T("Deobfuscate corrected the extension of %d file(s)") % nr_ext_renamed)
         filelist = newlist
 
+    """
```

Comment:
Using triple quotes is usually placed directly under the `def` to explain the function. I would suggest to move this there to.